plugins/auth-backend/src/lib/passport/PassportStrategyHelper.ts

```diff
@@ -95,8 +95,14 @@ export const executeFrameHandlerStrategy = async <T, PrivateInfo = never>(
       ) => {
         reject(new Error(`Authentication rejected, ${info.message ?? ''}`));
       };
-      strategy.error = (error: Error) => {
-        reject(new Error(`Authentication failed, ${error}`));
+      strategy.error = () => {
```

Comment:
So, why I asked is because you removed the error parameter here. And instead of printing the dynamic error message, you always assume that the error is about invalid credentials. I don't know if that is true or if the error could be something else. But if it can, that should be reflected in the message to the user.

Reply:
I totally got your point. I am not sure in that case. Closing this PR for better approach.